Loop/Managers/Store Protocols/GlucoseStoreProtocol.swift

```diff
@@ -26,6 +26,9 @@ protocol GlucoseStoreProtocol: AnyObject {
     
     var sharingDenied: Bool { get }
     
+    /// The length of time to delay until storing samples into HealthKit. 0 = no delay.
+    var healthKitStorageDelay: TimeInterval { get set }
```

Comment:
Does this need to be added? I think GlucoseStoreProtocol is used by LDM, but not DDM, right? DDM holds a reference to the actual GlucoseStore, I think, and that is the only place it looks like healthKitStorageDelay is being set on.

Reply:
Strictly speaking, you're right, I could just get rid of this.  But is it not the case that `GlucoseStoreProtocol` was introduced as a testing seam?  That's the only reason I put it there.  Again, technically speaking, `DeviceDataManager`'s unit tests would mock this for testing.  Unfortunately, I don't see unit tests for `DeviceDataManager` (?), so I guess I can get rid of this for now.

Reply:
Yes, the protocol is a (clumsy) seam for testing, that boxes up most of the GlucoseStore implementation. It would be better to have protocols that specify the roles (data provider for forecasts, effects calculation, and storage methods) that GlucoseStore plays, but we did a quick and dirty to get LDM under test coverage. The addition only struck me because I already feel bad about it being such a large mixed protocol.